src/user/introduction.rst

```diff
@@ -35,19 +35,19 @@ This came with a list of disadvantages:
 
 conda-forge is a community effort that tackles these issues:
 
- - all packages are shared in a single channel named ``conda-forge``
- - care is taken that all packages are up-to-date
- - common standards ensure that all packages have compatible versions
- - by default, we build packages for macOS, Linux AMD64 and Windows AMD64
- - many packages are updated by multiple maintainers with an easy option to become a maintainer
- - an active core developer team is trying to also maintain abandoned packages
+ - all packages are shared in a single channel named ``conda-forge``.
+ - care is taken that all packages are up-to-date.
+ - common standards ensure that all packages have compatible versions.
+ - by default, we build packages for macOS, Linux AMD64 and Windows AMD64.
+ - many packages are updated by multiple maintainers with an easy option to become a maintainer.
+ - an active core developer team is trying to also maintain abandoned packages.
 
 
 
 How can I install packages from conda-forge?
 --------------------------------------------
 
-Using conda-forge is as easy! Make sure you have at least ``conda >=4.6``,
+Using conda-forge is easy! Make sure you have at least ``conda >=4.6``,
```

Comment:
@conda-forge/core What minimum version do we need these days? It might be closer to 4.8 or 4.9 at this point?

Reply:
4.9 has the "Virtual package requirements are now considered by the solver" fix. I would say that is the minimum for conda-forge packages to work properly.

Reply:
Alright! :)
Also, I think we don't need the 'at least' when we are writing 'conda >=4.9'. What do you think?

Reply:
It should be:

Using conda-forge is easy! Make sure you have ``conda >=4.9``,

Reply:
I prefer the former but English is not my first language so I'm not sure if that reads better or not for a native speaker.

Reply:
@ocefpaf 
Both are grammatically correct. 
And the first one does sound better!